website/docs/d/pi_instance.html.markdown

```diff
@@ -75,3 +75,9 @@ In addition to all argument reference list, you can access the following attribu
 - `storage_type` - (String) The storage type where server is deployed.
 - `virtual_cores_assigned` - (Integer) The virtual cores that are assigned to the instance.
 - `volumes` - (List of strings) The list of volume IDs that are attached to the instance.
+
+  **Notes** Ibmi software licenses for IBMi virtual server instances -- only for IBMi instances
+- `pi_ibmi_css` - (Boolean) IBMi Cloud Storage Solution.
+- `pi_ibmi_pha` - (Boolean) IBMi Power High Availability.
+- `pi_ibmi_rds` - (Boolean) IBMi Rational Dev Studio.
+- `pi_ibmi_rds_users` - (Integer) IBMi Rational Dev Studio Number of User Licenses.
```

Comment:
Arrange this in alphabetical order as you did in the resource doc.

Reply:
Updated documentation